tests/unit/einstein/test_ceph.py

```diff
@@ -0,0 +1,177 @@
+import time
+import unittest
+import subprocess
+from ims.common import config
+
+config.load()
+
+import rados
+import rbd
+from ims.common import shell
+from ims.common import constants
+from ims.einstein import ceph
+from ims.einstein.operations import BMI
+from ims.exception import file_system_exceptions
+from ims.common.log import trace
+import logging
+
+_cfg = config.get()
+
+POOL = _cfg.fs.pool
+CEPH_IMG = "BMI_TEST_CEPH"
+TEST_NOT_EXIST_IMG_NAME = "BMI_TEST_IMG_NOT_FOUND"
+CEPH_IMG_SIZE = 1024
+RANGE_OUTSIDE_THE_IMAGE = 1025
```

Comment:
The name is not clear, this value is not a range. Think of a better name like index outside image or something. 
Also assign it to CEPH_IMG_SIZE + 1 as people will forget to change otherwise.